### ISSUE TITLE
Document Titles are not Searched

### ISSUE THREAD

**Issue by sulekz** [vectara/vectara-docs#160]
I'm using the search bar on [Vectara Docs site](docs.vectara.com) and noticed that while I can search for the contents of a page, I can't search for headings. For example in the screenshot below I'm searching for "File Upload API" which is one of the exact headings for a page, but its not showing up.

![slack_image](https://github.com/vectara/vectara-docs/assets/108358100/4e260376-389f-4bf9-a9fa-dde04b66ea21)

**Comment by pwoznic** [comment]
I want to test a theory.. PR 141 adds a new topic about managing documents in the UI and I also updated the topics to have a sidebar_label attribute instead of just a title. Older topics in this docset don't have sidebar_label while newer topics did (and so I've been adding it to new topics I've created). Anyway, I'm curious if it was searching on the sidebar title because I tested other topics and had similar behavior.
